src/models/preparation.js

```diff
@@ -49,7 +55,11 @@ module.exports = (sequelize, DataTypes) => {
 
   Preparation.associate = function(models) {
     this.belongsTo(models.Recipe, { foreignKey: 'recipeId' });
-    this.belongsTo(models.User, { foreignKey: 'userId' });
+    this.belongsTo(models.UserProfile, {
```

Comment:
Any reason not to keep the User association?

Reply:
Added back; I removed it, because I thought it would be safer to associate to the user profile than the user account.

Reply:
Oh I see, that makes sense. I think having both and only `include`ing the associations we need in find calls should do an OK job mitigating any potential for disclosure of information, but it is something to keep thinking about.